bindings/node/README.md

```diff
@@ -1,9 +1,19 @@
-MongoDB Client Encryption
-=========================
+# MongoDB Client Encryption
 
 The Node.js wrapper for [`libmongocrypt`](../../README.md)
 
-**Note** This library provides encryption functionality for the MongoDB Node.js driver but is **not intended** to be consumed in isolation.  The public API that uses the functionality in this library is available in the `mongodb` package.  We reserve the right to make breaking changes to `mongodb-client-encryption` that fall outside of semver.
+**Note** This library provides encryption functionality for the MongoDB Node.js driver but is **not intended** to be consumed in isolation. The public API that uses the functionality in this library is available in the `mongodb` package. We reserve the right to make breaking changes to `mongodb-client-encryption` that fall outside of semver.
+
+### MongoDB Node.js Driver Version Compatibility
+
+Only the following version combinations with the [MongoDB Node.js Driver](https://github.com/mongodb/node-mongodb-native) are considered stable.
+
+|               | `kerberos@1.x` | `kerberos@2.x` |
```

Comment:
@dariakp This is the wrong table, no?

Reply:
Yeah this is the kerberos table.

Reply:
This is what I get for having 6 vs code windows open with README files in each 😅 Fixed